core/src/commonMain/kotlin/dev/fritz2/identification/id.kt

```diff
@@ -1,8 +1,20 @@
 package dev.fritz2.identification
 
+object Id {
+    private const val defaultLength = 6
+    private val chars = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz".toCharArray()
+
+    fun next(length: Int = defaultLength) = buildString {
+        for (i in 0 until length) {
```

Comment:
How about `repeat(length) { append(chars.random()) }` ? Not sure whether your version might be faster though!

Reply:
I would prefer the for loop ;)